jetty-core/jetty-rewrite/src/main/java/org/eclipse/jetty/rewrite/handler/Rule.java

```diff
@@ -55,4 +58,116 @@ public String toString()
     {
         return "%s@%x[terminating=%b]".formatted(getClass().getSimpleName(), hashCode(), isTerminating());
     }
+
+    /**
+     * A tuple of a {@link Request.Wrapper} and is also a {@link org.eclipse.jetty.server.Request.Processor},
```

Comment:
It's not really a tuple like a Java `record` is. Maybe just "A Request.Wrapper that is also a Request.Processor".